Print_Odd_Numbers.py

```diff
@@ -0,0 +1,13 @@
+#Create a python program to print odd numbers
```

Comment:
This program is not working as expected. Please correct.

<img width="623" alt="image" src="https://user-images.githubusercontent.com/38807024/200739978-1b24fffb-aa45-4fc5-b9d3-169ebde44fd9.png">